github-releases.ts

```diff
@@ -25,42 +25,84 @@ interface GitHubReleaseEntry {
   draft: boolean
 }
 
+interface QuayResponse {
+  has_additional: boolean
+  page: number
+  tags: QuayImageTagEntry[]
+}
+
+interface QuayImageTagEntry {
+  name: string
+  reversion: boolean
+  end_ts: number
+  start_ts: number
+  image_id: string
+  last_modified: string
+  expiration: string
+  manifest_digest: string
+  docker_image_id: string
+  is_manifest_list: boolean
+  size: number
+}
+
 export const githubReleases = async () => {
-  fetch('https://api.github.com/repos/hedgedoc/hedgedoc/releases')
-    .catch(err => {
-      console.log("Can't connect to GitHub API ", err);
-      process.exit();
-    })
+  fetch('https://quay.io/api/v1/repository/hedgedoc/hedgedoc/tag/')
     .then(res => res.json())
-    .then((json: GitHubReleaseEntry[]) => {
-      json = json.filter((entry: GitHubReleaseEntry) => !entry.prerelease && !entry.draft)
+    .then((quayJson: QuayResponse) => {
+      const dockerTags = new Map<string, QuayImageTagEntry>()
+      quayJson.tags.forEach(tagEntry => {
+        dockerTags.set(tagEntry.name, tagEntry)
+      })
+      fetch('https://api.github.com/repos/hedgedoc/hedgedoc/releases')
+        .then(res => res.json())
+        .then((githubJson: GitHubReleaseEntry[]) => {
+          githubJson = githubJson.filter((entry: GitHubReleaseEntry) => !entry.prerelease && !entry.draft)
+
+          githubJson.forEach(entry => {
+            const dockerTag = dockerTags.get(entry.tag_name)
 
-      json.forEach(entry => {
-        const frontmatterJson = {
-          Title: entry.name,
-          date: entry.created_at,
-          tarball: entry.tarball_url,
-          assets: entry.assets.map(asset => {
-            return {
-              name: asset.name,
-              url: asset.browser_download_url,
-              size: asset.size
-            }
-          }),
-          githubLink: entry.html_url
-        };
+            const frontmatterJson = {
+              Title: entry.name,
+              date: entry.created_at,
+              tarball: entry.tarball_url,
+              docker: dockerTag
+                ? {
+                  tag: dockerTag.name,
+                  size: dockerTag.size,
+                  alpine: dockerTags.has(`${entry.tag_name}-alpine`)
```

Comment:
just `alpine` is a bit confusing, because it could also mean "this is an alpine image". Please use a name that is a bit more specific.

Reply:
Done